go-controller/pkg/cluster/master.go

```diff
@@ -309,7 +309,9 @@ func (cluster *OvnClusterController) SetupMaster(masterNodeName string) error {
 
 	// Create a logical switch called "join" that will be used to connect gateway routers to the distributed router.
 	// The "join" will be allocated IP addresses in the range 100.64.1.0/24.
-	stdout, stderr, err = util.RunOVNNbctl("--may-exist", "ls-add", "join")
+	stdout, stderr, err = util.RunOVNNbctl("--may-exist", "ls-add", "join",
+		"--", "set", "logical_switch", "join", "other-config:subnet=100.64.1.0/24",
+		"--", "set", "logical_switch", "join", "other-config:exclude_ips=100.64.1.1")
 	if err != nil {
```

Comment:
Also, at L330 below you don't need connect_to_join anymore since that was used to generate IPs in the old logic

```
 		routerMac = util.GenerateMac()
 		stdout, stderr, err = util.RunOVNNbctl("--", "--may-exist", "lrp-add", OvnClusterRouter,
 			"rtoj-"+OvnClusterRouter, routerMac, "100.64.1.1/24", "--", "set", "logical_router_port",
"rtoj-"+OvnClusterRouter, "external_ids:connect_to_join=yes")
```

Reply:
> Should we make this Subnet prefixlen configurable? With /24 we will only be able to create 253 nodes K8s cluster? What if we need more than those many number of nodes?

@girishmg I think that's a good idea, but it was a /24 before as well and I wasn't going to change that yet. Is that OK?

Reply:
Ok thanks.  Will do that separately.